taps_to_cases/runner/src/main/java/com/microsoft/hydralab/t2c/runner/T2CAppiumUtils.java

```diff
@@ -232,6 +232,14 @@ public static void chooseActionType(BaseDriverController driver, ActionInfo acti
                 isReset = (Boolean) arguments.getOrDefault("isReset", false);
                 driver.inspectMemoryUsage(targetApp, description, isReset);
                 break;
+            case "inspectLatencyStart":
```

Comment:
```suggestion
            case "inspectEventStart":
```

Reply:
Maybe this can be generalized as collecting the start and end of a event or process, inspectEventStart or inspectProcessStart both could work.

Reply:
Updated